src/shared/logger/activation.ts

```diff
@@ -43,7 +43,7 @@ export async function activate(
         {
             logPaths: [logUri.fsPath],
             outputChannels: [chan],
-            useConsoleLog: true,
+            useDeveloperToolsConsole: isInBrowser(),
```

Comment:
This is an OK compromise for now, but ideally this would be decided by `src/extensionWeb.ts` , right?

Reply:
Yeah this is a compromise for now. We'd need to add a new argument to [`activate()`](https://github.com/aws/aws-toolkit-vscode/blob/master/src/shared/logger/activation.ts#L29) but there are different ways to do it which I didn't want to figure out in this PR